radixdlt-java/src/main/java/com/radixdlt/client/application/translate/TokenTransferTranslator.java

```diff
@@ -176,12 +177,13 @@ public TokenTransferTranslator(RadixUniverse universe) {
 				encryptor.getProtectors().stream().map(EncryptedPrivateKey::base64).forEach(protectorsJson::add);
 
 				byte[] encryptorPayload = protectorsJson.toString().getBytes(StandardCharsets.UTF_8);
-				StorageParticle encryptorParticle = new StorageParticleBuilder()
+				MessageParticle encryptorParticle = new MessageParticleBuilder()
 						.payload(encryptorPayload)
 						.setMetaData("application", "encryptor")
 						.setMetaData("contentType", "application/json")
 						.account(transfer.getFrom())
```

Comment:
Should not need this as MessageParticleBuilder should "know" that source should be one of the accounts to add.